danger-kotlin-sdk/src/main/kotlin/systems/danger/kotlin/sdk/DangerKotlinAPI.kt

```diff
@@ -17,22 +86,66 @@ interface DangerContext {
     val markdowns: List<Violation>
 }
 
+/**
+ * Violation is any comment on your Pull Request
+ *
+ * @param message the violation message
+ * @param file the path to the target file
+ * @param line the line number into the target file
+ * @constructor Create empty Violation
+ */
 data class Violation(
     val message: String,
     val file: String? = null,
     val line: Int? = null
 )
 
+/**
+ * Describe the Sdk, contains:
+ * - [Sdk.API_VERSION]
+ * - [Sdk.VERSION_NAME]
+ *
+ * @constructor Create empty Sdk
```

Comment:
Does anyone need to create an SDK object? Is there a way to make this impossible?

Reply:
Is a singleton with two fields, in this case works like a static class with const fields 😄 this is how works.
using object in kotlin is also a way to group under the same context static fields